catalog/USGS/USGS_NLCD_RELEASES_2021_REL_NLCD.jsonnet

```diff
@@ -0,0 +1,462 @@
+               from the 2018 NavStreets Street Data.
+            |||,
+          },
+          {
+            value: 22,
+            color: '0000ff',
+            description: |||
+              Tertiary road. Any two-lane road. Pixels were derived from the
+              2018 NavStreets Street Data.
+            |||,
+          },
+          {
+            value: 23,
+            color: 'ffffff',
+            description: |||
+              Thinned road. Small tertiary roads that generally are not paved
+              and have been removed from the landcover but remain as part of
+              the impervious surface product. Pixels were derived from the 2018
+              NavStreets Street Data.
+            |||,
+          },
+          {
+            value: 24,
+            color: 'ffc0c5',
+            description: |||
+              Non-road non-energy impervious. Developed areas that are
+              generally not roads or energy production; includes
+              residential/commercial/industrial areas, parks, and golf courses.
+            |||,
+          },
+          {
+            value: 25,
+            color: 'eb82eb',
+            description: |||
+              Microsoft buildings. Buildings not captured in the NLCD
+              impervious process, and not included in the nonroad impervious
+              surface class. Pixels derived from the Microsoft US Building
+              Footprints dataset.
+            |||,
+          },
+          {
+            value: 26,
+            color: '9f1feb',
+            description: |||
+              LCMAP impervious. Impervious pixels from LCMAP that were used
+              to fill in gaps left when roads were updated from previous
+              versions of NLCD.
+            |||,
+          },
+          {
+            value: 27,
+            color: '40dfd0',
+            description: |||
+              Wind turbines. Pixels derived from the US Wind Turbine Dataset,
+              accessed on 1/9/2020.
+            |||,
+          },
+          {
+            value: 28,
+            color: '79ff00',
+            description: |||
+              Well pads. Pixels derived from the 2019 Oil and Natural Gas Wells
+              dataset from the Oak Ridge National Laboratory.
+            |||,
+          },
+          {
+            value: 29,
+            color: '005f00',
+            description: |||
+              Other energy production. Areas previously identified as well pads
+              and wind turbines and classified in coordination with the
+              Landfire project.
+            |||,
+          },
+        ],
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'Landcover',
+        lookat: {
+          lat: 38.686,
+          lon: -115.356,
+          zoom: 5,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              0.0,

```

Comment:
No need to include .0 for integer values. Here and below.

Reply:
updated